### PR TITLE
chore: Added Unit tests to FileSystemDeleteHandler

### PR DESCRIPTION
**Pull request by derektriley**
**Description**:
Adds unit tests to FileSystemDeleteHandler. Brings coverage to 92%.

**Related issue(s)**:
#13192 

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/handlers/FileSystemDeleteTest.java [hunk 1]
```diff
@@ -21,19 +21,23 @@
 import static com.hedera.hapi.node.base.ResponseCodeEnum.UNAUTHORIZED;
 import static com.hedera.node.app.spi.fixtures.Assertions.assertThrowsPreCheck;
 import static com.hedera.test.utils.KeyUtils.A_COMPLEX_KEY;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
```
Comment (thomas-swirlds-labs) [inline hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/handlers/FileSystemDeleteTest.java:24]: We should standardize on using AssertJ over Junit/Jupiter Assertions

Comment (povolev15) [reply hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/handlers/FileSystemDeleteTest.java:24]: As we mentioned in other PR., that's supposed to be our standard.

**Review by povolev15 — DISMISSED** [review]
TY @derektriley LGTM just change to AssertJ please

**Review by povolev15 — APPROVED** [review]
LGTM TY @derektriley

**Review by thomas-swirlds-labs — APPROVED** [review]

**Review by Neeharika-Sompalli — APPROVED** [review]
LGTM! Thanks @derektriley

**Pull request merged**